Resources/Prototypes/_LostParadise/Entities/Objects/Specific/Cyborgs/borg_modules_syndi_medical.yml

```
@@ -0,0 +1,86 @@
+    - LPPBorgModuleSyndicateMedical
+
+- type: entity
+  id: LPPBorgModuleHealing
+  parent: [ LPPBaseBorgModuleSyndicateMedical, BaseProviderBorgModule ]
+  name: syndicate medical cyborg module
+  description: A module with healing staff.
+  components:
+  - type: Sprite
+    layers:
+    - state: syndicate
+    - state: icon-diagnosis #Заменить
```

yamllint (1.35.1)

[warning] too few spaces before comment

(comments)